master/getting-started/bare-metal/bare-metal.md

```diff
@@ -666,10 +715,8 @@ EOF
 > stack where it is not yet determined what a packet's outgoing interface will
 > be.)
 >
-> Because these are normal host endpoint policies, they are not enforced for
-> traffic that is sent from a local pod.  Calico does not yet have a form of
-> host endpoint protection that can be used to restrict outbound traffic from a
-> local workload.
+> Because these are normal host endpoint policies which do not apply to forward traffic (`applyOnForward` is `false`), they are enforced for
+> traffic that is from a local process.  
```

Comment:
Better to revert to "... they are not enforced for traffic that is sent from a local pod."  Because there is no other mention of local processes in this example.

(But good to delete "Calico does not yet have a form ...".)